rq_dashboard/web.py

```diff
@@ -267,17 +240,24 @@ def list_instances():
 @blueprint.route('/queues.json')
 @jsonify
 def list_queues():
-    queues = serialize_queues(sorted(get_all_queues()))
+    queues = serialize_queues(sorted(Queue.all()))
     return dict(queues=queues)
 
 
 @blueprint.route('/jobs/<queue_name>/<page>.json')
 @jsonify
 def list_jobs(queue_name, page):
     current_page = int(page)
-    queue = get_queue(queue_name)
+    registry_swapper = '-failed' in queue_name
```

Comment:
Better to implement as `/jobs/<queue_name>/registries/<registry_name>/<page>.json`

Reply:
Implemented.